start-proxy/action.yml

```
@@ -6,6 +6,9 @@ inputs:
     description: The URLs and credentials of package registries
     required: false
     default: "[]"
+  registries_credentials:
+    description: Base64 encoded JSON configuration for the URLs and credentials of the package registries
```

Is there a link or some more information you can put here? Otherwise it's opaque to users how this input is structured. Since this is still an experimental action, not high priority right now.

---

I don't think users will need to use this input. Is there a way to denote it as "internal"?

---

Maybe just doing this?

```suggestion
    description: [Internal] Base64 encoded JSON configuration for the URLs and credentials of the package registries
```

Am I correct in that it's unlikely users will be directly accessing this action?

---

Yes, the entire action is meant to be internal. I'll add that to the description.